src/pip/_internal/index/package_finder.py

```diff
@@ -221,9 +221,8 @@ def evaluate_link(self, link):
             ignore_requires_python=self._ignore_requires_python,
         )
         if not supports_python:
-            # Return None for the reason text to suppress calling
-            # _log_skipped_link().
-            return (False, None)
+            reason = f'{version} requires-python {link.requires_python}'
```

Comment:
Probably want to use `Requires-Python` instead since that’s the casing used in the standard.

Reply:
okay, will fix.